docs/env-variables.md

```diff
@@ -0,0 +1,27 @@
+# Different Environment Variables used
+
+This page describes about the different environment variables that are
+used to configure kube-scheduler-simulator.
+
+`Port`: This is the port number on which kube-scheduler-simulator is started and it's value is 1212 which is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/3111ec419ccb8c49197b75385fbd166f7f159435/docker-compose.yml#L7)
+
+`EtcdURL`: This is the URL for kube-scheduler-simulator etcd which runs locally and it's value is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/3111ec419ccb8c49197b75385fbd166f7f159435/docker-compose.yml#L8)
+
+`FrontendURL`: This URL is on which kube-scheduler-simulator is reachable locally and it's value is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/3111ec419ccb8c49197b75385fbd166f7f159435/docker-compose.yml#L9).
+
+`KUBECONFIG`: Currently not used. [Beta: Existing Cluster Importing]
+This variable is used to find the information it needs to access the
+cluster which will be imported to our simulator and communicate with
+the API server of the cluster. If you use the Beta feature, you need
+to set the `Kubeconfig` file path of the cluster which will be
+imported. [Read more]. [Read
+more](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#the-kubeconfig-environment-variable)
```

Comment:
```suggestion
`KUBECONFIG`: This is for the beta feature "Existing Cluster Importing" 
and the feature is enabled when you set the value.
Currently, the simulator tries to import ALL resources related to scheduling from your cluster,
so, please be careful (or avoid) using it in clusters with large amounts of resources. 
This variable is used to find KubeConfig it needs to access your
cluster to import resources to our simulator. 
```

Reply:
Made a small change in the last two lines: `Kubeconfig required to access your cluster for importing resources to scheduler simulator`. Let me know if this looks good.